content/deployment/install.md

```diff
@@ -209,304 +147,85 @@ Once you downloaded and verified the release, you are ready to continue with the
+- By default [SecDefaultAction](https://github.com/SpiderLabs/ModSecurity/wiki/Reference-Manual-%28v2.x%29#SecDefaultAction) CRS will redirect to your local domain when an alert is triggered. This may cause redirect loops depending on your configuration. Take some time to decide what you want ModSecurity it do (drop the packet, return a status:403, go to a custom page etc.) when it detects malicious activity.
+- Make sure to configure your anomaly scoring thresholds for more information see [Anomaly]({{< ref "anomaly_scoring.md" >}} "Anomaly")
+- By default our rules will consider many issues with different databases and languages. If you are running in a specific environment, you probably want to limit this behavior for performance reasons.
+- ModSecurity supports [Project Honeypot](https://www.projecthoneypot.org/) blacklists. This is a great project and all you need to do to leverage it is sign up for an [API key](https://www.projecthoneypot.org/httpbl_api.php) (⚠️ it might be outdated)
+- Do make sure you have added any methods, static resources, content types, or file extensions that your site needs beyond the basic ones listed.
+
+For more information please see the page on [configuration]({{< ref "crs.md" >}} "configuration"). Once you have reviewed
+and configured CRS you should rename the file suffix from `.example` to `.conf`:
 
 ```bash
 mv csr-setup.conf.example csr-setup.conf
 ```
 
-In addition to `csr-setup.conf.example` there are two other .example files
-within our repository. These files are:
-`rules/REQUEST-00-LOCAL-WHITELIST.conf.example` and
-`rules/RESPONSE-99-EXCEPTIONS.conf.example`. These files are designed to
-provide the rule maintainer the capability to modify rules (see
-[false positives and tuning]({{< ref "../configuring/false_positives_tuning.md" >}}#rule-exclusions)) without breaking forward
-compatability with updates. As such you should rename these two files,
-removing the `.example` suffix. This will make it so that even when
-updates are installed they do not overwrite your custom updates. To
-rename the files in Linux one would use a command similar to the
+In addition to `csr-setup.conf.example` there are two other .example files within our repository. These files are:
+`rules/REQUEST-900-EXCLUSION-RULES-BEFORE-CRS.conf.example` and `rules/RESPONSE-999-EXCLUSION-RULES-AFTER-CRS.conf.example`. These files are designed to provide the rule maintainer the capability to modify rules (see [false positives and tuning]({{< ref "../configuring/false_positives_tuning.md" >}}#rule-exclusions)) without breaking forward compatibility with updates. As such you should rename these two files, removing the `.example` suffix. This will make it so that even when updates are installed they do not overwrite your custom updates. To rename the files in Linux one would use a command similar to the
```

Comment:
If this is so central, why don't we just ship the files without the `example` extension? Can you explain that?

Reply:
Well, we need to ask why this was designed this way. Probably so it doesn't override the file of already present.

Reply:
@dune73?